x-pack/plugins/transform/common/api_schemas/transforms.ts

```diff
@@ -58,6 +58,7 @@ export const settingsSchema = schema.object({
 });
 
 export const sourceSchema = schema.object({
+  runtime_mappings: schema.maybe(schema.recordOf(schema.string(), schema.any())),
```

Comment:
Could you please improve the validation? I think we can replace `schema.any()` with `schema.object()` and describe it

Reply:
Updated here [`29778b2` (#90363)](https://github.com/elastic/kibana/pull/90363/commits/29778b2a10f63947935f2dd405f8b8d5785e98cf)